README.md

```diff
@@ -1,2 +1,28 @@
 # Hermes 
+
+**This is not an officially supported Google product.**
+
 Hermes is an awesome project!
```

Comment:
Although this is 100% true, lets replace this with a TODO to add more information on how to bring up hermes and how to get started with it.

Reply:
I've addressed this with a suggestion and new commit.